raiden_contracts/tests/utils.py

```diff
@@ -156,6 +156,87 @@ def get_settlement_amounts(
         participant2_locked=participant2_locked,
     )
 
+def get_onchain_settlement_amounts(
```

Comment:
Please use this in `settle_state_tests`, so we can compare both `get_onchain_settlement_amounts` and `get_settlement_amounts` and make sure there is no difference / no failing tests.
https://github.com/raiden-network/raiden-contracts/blob/40197cc59c61065c8d33eb2cefa35123e27b91fe/raiden_contracts/tests/fixtures/channel.py#L354